tests/app/broadcast_message/test_rest.py

```diff
@@ -24,6 +24,8 @@ def test_get_broadcast_message(admin_request, sample_broadcast_service):
     )
 
     assert response['id'] == str(bm.id)
+    assert response['template_id'] == str(t.id)
+    assert response['template_content'] == t.content
```

Comment:
Should we be checking the fact that the content is rendered with placeholders filled in? I'm guessing the template in this case doesn't have any placeholders so the test passes regardless though? And I see another PR which is going to stop us from allowing placeholders in it so maybe we should instead change `def content` such that it doesn't try and fill in placeholders?

Or should we be

Reply:
I’ve added a test in 80352baf

Reply:
Cool, this is good. Thanks